integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html

```diff
@@ -14,15 +14,25 @@
 <ng-template #paymentData let-paymentLink="paymentLink">
   <div>
     <div
-      *ngIf="paymentLink.renderType === RENDER_TYPES.DESTINATION"
+      *ngIf="paymentLink.renderType === RENDER_PATTERN.FULL_PAGE"
       class="cx-payment-link"
     >
-      <a
-        *ngIf="!paymentLink.destination?.form?.length"
-        class="btn btn-primary"
-        [href]="paymentLink.data"
-        >{{ 'opf.checkout.proceedPayment' | cxTranslate }}</a
-      >
+      <a class="btn btn-primary" [href]="paymentLink.data">{{
+        'opf.checkout.proceedPayment' | cxTranslate
+      }}</a>
+    </div>
+    <ng-container
+      *ngIf="
+        paymentLink.data && paymentLink.renderType === RENDER_PATTERN.IFRAME
```

Comment:
do we also need to cover the use case where renderType is IFRAME and based on script (not url). In my understanding it was the use case we were covering before this fix.